github_changelog_generator.gemspec

```diff
@@ -27,6 +27,9 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "github_api", ">= 0.12"
   spec.add_runtime_dependency "rainbow", ">= 2.1"
 
+  # rack 2.0 does not work on Ruby < 2.2.  Lock it down here
+  spec.add_runtime_dependency "rack", "~> 1.2"
```

Comment:
This is contentious: if people are using this library in their application, we're holding the rack back.